src/c++/perf_analyzer/data_loader.cc

```diff
@@ -339,43 +339,35 @@ DataLoader::GetInputData(
     // Get the data and the corresponding byte-size
     auto it = input_data_.find(key_name);
     if (it != input_data_.end()) {
+      data.is_valid = true;
```

Comment:
Why is the check moved up here?

Reply:
Which check? The setting of `valid` to true? It was always set to true in this code block, just at the end instead of the start. No logical change

Reply:
Sorry, I meant to say assignment.